node/hack/governor/src/index.ts

```diff
@@ -274,7 +276,7 @@ axios
 
             // We add in the "=" character to ensure an undefined symbol
             // does not mess up the removed tokens logic
-            newTokenKeys[chain + "-" + wormholeAddr] = "=" + data.Symbol;
+            newTokenKeys[chain + "-" + wormholeAddr] = ["=" + data.Symbol, data.CoinGeckoId];
```

Comment:
If `data.Symbol` is undefined, is this line going to hide that? It might make it look like `data.CoinGeckId` is the symbol.
It might be worth replacing `data.Symbol` with a string like `UNDEFINED` manually if it doesn't exist rather than use an empty string (which I assume is what happens now)

Reply:
I might be misunderstanding but I don't think this is true? The symbol logic is identical to before, I've just changed the mapping from `string=>string` to `string=>[string, string]`. And I index into the array with the changes below

Reply:
There are a few entries above that look like this:

```
 {
  "token": "1-05beb884b642b260d9c2b53cb3d876b8b4de72aea0d787d779bf4266688a3935-",
  "previousPrice": 14.81,
  "newPrice": 0.00451702,
  "percentageChange": "-100.0"
 },
 ```
 
 Where the token has no Symbol and just ends with `-`. I think in this case we just know, "ah, there's no symbol." But if there is only the Symbol and no CG ID, or vice-versa, this might get weird.
 
 Anyway it's a small detail so we can drop it here and maybe pick it up another time.

Reply:
No symbol is a valid case for SPL tokens, not sure if that's still true for new tokens, but at least historically that was possible which is why we have those. Re CoinGecko ID, I think we're guaranteed to have that because otherwise we wouldn't have the data being returned from the API in the first place. Plus here's an example of how it would render:

```
 {
  "token": "1-067d3988ce882945c63a0717a92538addd9a37550ce5cd711f76ca74a4c89786-",
  "previousPrice": 0.00016403,
  "newPrice": 0.00010722,
  "percentageChange": "-34.6",
  "url": "https://www.coingecko.com/en/coins/solcial"
 },
```

Reply:
OK cool, thanks for walking me through that